src/content/docs/zh-cn/guides/integrations-guide/cloudflare.mdx

```diff
@@ -72,20 +72,107 @@ export default defineConfig({
 
 要为每个页面编译一个单独的 bundle，请在 Cloudflare 适配器配置中设置 `functionPerPath` 选项。该选项需要手动维护 `functions` 文件夹。Astro 发出的文件将覆盖具有相同名称的现有 `functions` 文件，因此你必须为手动添加的每个文件选择唯一的文件名。此外，适配器永远不会清空 `functions` 文件夹中的过时文件，因此当你删除页面时，必须手动清理该文件夹。
 
-```diff
+```js ins={8}
+// astro.config.mjs
 import {defineConfig} from "astro/config";
 import cloudflare from '@astrojs/cloudflare';
 
 export default defineConfig({
      adapter: cloudflare({
         mode: 'directory',
-+       functionPerRoute: true
+        functionPerRoute: true
```

Comment:
I recommend continuing to use the normal diff syntax here, I might in the future take care of this conversion automatically within the script and then I'd be fine with having integrations with the more complex Expressive Code cases like this.

Reply:
ok. Just let me revert them.